### ISSUE TITLE
How to predict as performant as possible

### ISSUE THREAD

**Issue by jschneider** [openvinotoolkit/anomalib#1917]
Hey, 

we are using this code, which works fine:

```
dataset = PredictDataset(image_path)
dataloader = DataLoader(dataset, num_workers=15)

model = get_model("Patchcore")

output = engine.predict(
  model=model,
  ckpt_path=model_path,
  dataloaders=[dataloader],
)
```

The output contains everything I expect, everything is fine.

But:
I have a lot of pictures, I want to analyze - and performance matters:
* Therefore, it would be great if I was able to load the model only once and use it for multiple images.
* It would be great to avoid generating the debug picture (containing the heat map, mask etc).

I am sure I am missing a simple way to achieve this. Any help appreciated ;-)

**Comment by JohannesSteinle** [comment]
I have had the same problem and did not find a fix. Looking forward to a solution!

**Comment by samet-akcay** [comment]
it is currently not configurable, but we will enable it in v1.1.0. For now, you could disable these lines to not visualize the outputs
https://github.com/openvinotoolkit/anomalib/blob/ca90807f0b3464e91dac765f33d574467f3d3353/src/anomalib/engine/engine.py#L412-L418

**Comment by jschneider** [comment]
Great! Thanks.

Will it be possible to load the model just once and call "predict" several times?

**Comment by samet-akcay** [comment]
yes, ideally it should be possible. Let us know if it does not work

**Comment by jschneider** [comment]
I can't find a way to load the model once and reuse the loaded model.
Could you give me a hint? I am quite new in this area, so might be a bit blind here...

**Comment by watertianyi** [comment]
@samet-akcay 
   from anomalib.visualization.image import ImageVisualizer
ModuleNotFoundError: No module named 'anomalib.visualization'

**Comment by samet-akcay** [comment]
@watertianyi, note that this has not been merged to `main` branch yet. It is in `feature/v2`, and will be released within v2, which is scheduled to be mid-December

**Comment by samet-akcay** [comment]
https://github.com/openvinotoolkit/anomalib/tree/feature/v2/src/anomalib/visualization

**Comment by watertianyi** [comment]
@samet-akcay 
If I want to use the visual function now, how should I adjust the code?

**Comment by samet-akcay** [comment]
Closing this as https://github.com/openvinotoolkit/anomalib/issues/2369 is the official ticket to track this
